package.json

```diff
@@ -21,6 +21,7 @@
   },
   "homepage": "https://github.com/serverless/serverless-kubeless#readme",
   "dependencies": {
+    "adm-zip": "^0.4.7",
```

Comment:
Actually they seem to use `jszip` that seems to be more popular. _Migrating to that module_